articles/azure-policy/azure-policy-introduction.md

```diff
@@ -39,7 +39,7 @@ Every policy definition has conditions under which it is enforced. And, it has a
 In Azure Policy, we offer some built-in policies that are available to you by default. For example:
 
 - **Require SQL Server 12.0**: This policy definition has conditions/rules to ensure that all SQL servers use version 12.0. Its effect is to deny all servers that do not meet these criteria.
-- **Allowed Storage Account SKUs**: This policy definition has a set of conditions/rules that determine if a storage account that is being deployed is within a set of SKU sizes. Its effect is to deny all servers that do not adhere to the set of defined SKU sizes.
+- **Allowed Storage Account SKUs**: This policy definition has a set of conditions/rules that determine if a storage account that is being deployed is within a set of SKU sizes. Its effect is to deny all Storage Accounts that do not adhere to the set of defined SKU sizes.
```

Comment:
@AErmie Please lower case 'storage accounts' and I'll approve this.  Thanks for catching this and contributing!  #in-progress